devicetypes/erocm123/inovelli-2-channel-smart-plug.src/inovelli-2-channel-smart-plug.groovy

```diff
@@ -252,13 +252,19 @@ private channelNumber(String dni) {
 }
 private void createChildDevices() {
 	state.oldLabel = device.label
-	for (i in 1..2) {
-		addChildDevice("Switch Child Device", "${device.deviceNetworkId}-ep${i}", null, [completedSetup: true, label: "${device.displayName} (CH${i})",
-																						 isComponent: true, componentName: "ep$i", componentLabel: "Channel $i"
-		])
+	try {
+		for (i in 1..2) {
+			addChildDevice("Switch Child Device", "${device.deviceNetworkId}-ep${i}", null, [completedSetup: true, label: "${device.displayName} (CH${i})",
+				isComponent: false, componentName: "ep$i", componentLabel: "Channel $i"
+			])
+		}
+	} catch (e) {
+		runIn(2, "sendAlert")
 	}
 }
-
+private sendAlert() {
```

Comment:
Why is this needed? The `Switch Child Device` is published by us so I'm not sure if this error would ever be hit.